vite.config.ts

```diff
@@ -23,7 +23,35 @@ export default defineConfig(() => {
             port: 8080,
             origin: 'http://localhost:8080'
         },
-        assetsInclude: ['**/*.wasm']
+        assetsInclude: ['**/*.wasm'],
+        optimizeDeps: [
+
+            'monaco-editor',
+            'vscode',
+            'vscode/extensions',
+            'vscode/services',
+            'vscode/monaco',
+            'vscode/service-override/model',
+            'vscode/service-override/editor',
+            'vscode/service-override/notifications',
+            'vscode/service-override/dialogs',
+            'vscode/service-override/configuration',
+            'vscode/service-override/keybindings',
+            'vscode/service-override/textmate',
+            'vscode/service-override/theme',
+            'vscode/service-override/languages',
+            'vscode/service-override/audioCue',
+            'vscode/service-override/views',
+            'vscode/service-override/quickaccess',
+            'vscode/service-override/debug',
+            'vscode/service-override/preferences',
+            'vscode/service-override/snippets',
+            'vscode/service-override/files',
+            'vscode/default-extensions/theme-defaults',
+            'vscode/default-extensions/javascript',
+            'vscode/default-extensions/typescript',
+            'vscode/default-extensions/json'
```

Comment:
Do you need to do that? I was thinking it's required only in the vscode-api demo because it's a local dependency